updater/config/.npmrc

```diff
@@ -1,13 +1,6 @@
 # TODO: Remove these hacks once we've deprecated npm 6 support as it no longer
 # spwans a child process to npm install git dependencies.
 
-# Only set our custom CA cert for npm because the system ca's + our custom ca
-# causes npm to blow up when installing git dependencies (E2BIG exception). This
-# happens because the ca-file contents are passed as a cli argument to npm
-# install from npm/cli/lib/pack.js as --ca="contents of ca file" - "ca" is
-# populated automatically by npm when setting "--cafile" and passed through in
-# when spawning the cli to install git dependencies.
-cafile=/usr/local/share/ca-certificates/dbot-ca.crt
```

Comment:
A note for the reviewers: That CA file does not appear in the dev-shell. While npm and yarn silently ignore its absence, Bun throws an error since it can't be found. Let me know if this is okay to remove, it appears all the tests continue to pass without this.